src/test/java/org/broadinstitute/consent/http/models/InstitutionDomainMapTest.java

```diff
@@ -0,0 +1,83 @@
+package org.broadinstitute.consent.http.models;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.google.gson.Gson;
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+public class InstitutionDomainMapTest {
+
+  @Test
+  public void testInstitutionDomainMapGetAndSet() {
+    InstitutionDomainMap map = new InstitutionDomainMap();
+
+    // should be null by default
+    assertNull(map.getInstitutionDomainMap());
+
+    // set a test map
+    Map<String, Set<String>> testMap = Map.of(
+        "Broad Institute", Set.of("broadinstitute.org", "broad.mit.edu"),
+        "Harvard", Set.of("harvard.edu", "hms.harvard.edu")
+    );
+    map.setInstitutionDomainMap(testMap);
+
+    assertEquals(testMap, map.getInstitutionDomainMap());
+    assertEquals(2, map.getInstitutionDomainMap().size());
+    assertEquals(2, map.getInstitutionDomainMap().get("Broad Institute").size());
```

Comment:
I find hamcrest better for this, but for some it can be harder to read (and if you're not familiar with it, harder to write), but you could instead write
```suggestion
    assertThat(map.getInstitutionDomainMap(), 
        allOf(aMapWithSize(2), hasEntry(is("Broad Institute"), hasSize(2))));
```